packages/params/test/e2e/ensure-config-is-synced.test.ts

```diff
@@ -25,12 +31,19 @@ describe("Ensure config is synced", () => {
 });
 
 function assertCorrectPreset(localPreset: BeaconPreset, remotePreset: BeaconPreset): void {
```

Comment:
Remote preset is missing `MAX_BLOBS_PER_BLOCK` since it has moved to config. But Lodestar has not made it configurable and it remains in local config. 

Adding  `ignoredLocalPresetFields` is a temporary workaround to make this test pass such that this PR is safe to be merged into unstable. 

See #7172 for details